### PR TITLE
style: add citation

### PR DESCRIPTION
**Pull request by Hao-Chalmers**
### Main improvements in this PR:
* Include new scientific publication that is conducted using Human-GEM

**I hereby confirm that I have:**

- [ ] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
I was expecting this citation in the other repos, not in Human-GEM.

**Comment by Hao-Chalmers** [comment]
@mihai-sysbio good idea, will add citation to other repos as well.

Note that the other repos are based on the Human-GEM template (with 99% similarity). And all modelling functions were developed and deposited in Human-GEM. They collectively serve as an integrated platform, rather than separated.

**Comment by JonathanRob** [comment]
As a user, if I'm looking at the repo to see how to cite using the resource, it's not clear to me which of these citations I should use. Since I think it's excessive to request that people always cite both papers, then I suggest including explicit instructions on when one and/or the other citation should be used.

**Comment by Hao-Chalmers** [comment]
Citation instructions were added as suggested by @JonathanRob

File: README.md [hunk 1]
```diff
@@ -7,10 +7,19 @@
...
+
+If you use Mouse1, Rat1, Zebrafish1, Fruitfly1 or Worm1 in your research, please cite:   
+
+ > H. Wang, J. L. Robinson, P. Kocabaş, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
+
+If you use Human1 in your research, please cite:  
 
  > J. L. Robinson, P. Kocabaş, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
 
+Starting from Human-GEM v1.5.0, all the releases are also archived in [Zenodo](https://doi.org/10.5281/zenodo.4099692) from which specific version can be cited if used.
```
Comment (mihai-sysbio) [inline README.md:20]: The ordering here is based on the most recent work. However, this being the repository for Human-GEM, I expect most visitors to be interested in reading about or citing Human1, so I am tempted to place that first - simply reordering without changing the text would be fine by me.

Comment (Hao-Chalmers) [reply README.md:20]: sounds good to me

File: README.md [hunk 2]
```diff
@@ -7,10 +7,19 @@
...
+
+If you use Human1 in your research, please cite:  
 
  > J. L. Robinson, P. Kocabaş, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
 
+ If you use Mouse1, Rat1, Zebrafish1, Fruitfly1 or Worm1 in your research, please cite:   
+
+  > H. Wang, J. L. Robinson, P. Kocabaş, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
+
+Starting from Human-GEM v1.5.0, all the releases are also archived in [Zenodo](https://doi.org/10.5281/zenodo.4099692) from which specific version can be cited if used.
```
Comment (mihai-sysbio) [inline README.md:20]: Would it make sense to move the line about Zenodo next to Human-GEM as well?

```suggestion
If you use Human1 in your research, please cite:  

 > J. L. Robinson, P. Kocabaş, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
 
Starting from Human-GEM v1.5.0, all the releases are also archived in [Zenodo](https://doi.org/10.5281/zenodo.4099692) from which specific version can be cited if used.

If you use Mouse1, Rat1, Zebrafish1, Fruitfly1 or Worm1 in your research, please cite:   

  > H. Wang, J. L. Robinson, P. Kocabaş, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)

```

**Review by mihai-sysbio — APPROVED** [review]
Good job on providing clear instructions on how to cite the foundational Human1 and the recent work in the model organism GEMs.

**Pull request merged**